web/src/popover_menus.ts

```diff
@@ -27,7 +27,8 @@ type PopoverName =
     | "change_visibility_policy"
     | "personal_menu"
     | "gear_menu"
-    | "help_menu";
+    | "help_menu"
+    | "stream_info_popover";
```

Comment:
Probably this `stream_info_popover` should be `stream_card_popover` to give this a consistent name?

Reply:
I did this rename, for this and `stream-info-popover`.